.coafile

```diff
@@ -14,7 +14,9 @@ language = python
 preferred_quotation = '
 
 [autopep8]
-bears = PEP8Bear, PycodestyleBear
+bears = PEP8Bear, PycodestyleBear, PyDocStyleBear
+pydocstyle_ignore = D100, D101, D102, D103, D104, D107,
+  D200, D203, D205, D209, D212, D213, D400
```

Comment:
Is this the preferred way to break long list of items in a coafile?

Reply:
Have a look at https://github.com/coala/coala/blob/master/.coafile#L33

Reply:
ya, that is fine.